### PR TITLE
Fix pie charts with empty metrics

### PR DESCRIPTION
**Pull request by KaitCrawford**

### CODE REVIEW

**Comment by KaitCrawford** [comment]
@miltontony @justinvdm ready for review

**Comment by KaitCrawford** [comment]
Broken charts:
Note how the bar chart has been affected by the error in the pie chart
broken_chart
Fixed charts:
Bar chart now displaying
fixed_metrics

File: ci/templates/ci/dashboard.html [hunk 1]
```diff
@@ -176,7 +184,11 @@ <h2 class="dashhead-title">{{ dashboard.name }} Dashboard</h2>
     metrics(widget)
       .forEach(function(d) {
         if (widget['type'] == 'pie'){
-          d.value = data[d.key][data[d.key].length-1].y;
+          if (data[d.key].length == 0) {
```
Comment (miltontony) [inline ci/templates/ci/dashboard.html]: you can simplify this by using the shorthand:
`d.value = data[d.key].length > 0 ? data[d.key][data[d.key].length-1].y : 0;`

**Comment by miltontony** [comment]
👍 thanks!

**Pull request merged**